src/tests/tests_runtime.rs

```diff
@@ -139,22 +142,29 @@ fn test_builtins() {
         "/wasm/use_builtins.wasm"
     ));
     let runtime_module = RuntimeModule::new(module, 200_000, gas_costs.clone()).unwrap();
+    let before = chrono::offset::Utc::now().timestamp_millis();
     match run_main(&*interface, runtime_module, 10_000_000, gas_costs.clone()) {
         Err(e) => {
-            println!("Error: {}", e);
-            assert!(e.to_string().starts_with(
-                "RuntimeError: Runtime error: error: abord with date and rnd at use_builtins.ts"
-            ));
+            let msg = e.to_string();
+            // make sure the error was caused by a manual abort
+            assert!(msg.contains("Manual abort"));
+            // check the given timestamp validity
+            let after = chrono::offset::Utc::now().timestamp_millis();
+            let ident = "UTC timestamp (ms) = ";
+            let start = msg.find(ident).unwrap_or(0).saturating_add(ident.len());
+            let end = msg.find(" at use_builtins.ts").unwrap_or(0);
+            let sc_timestamp: i64 = msg[start..end].parse().unwrap();
+            assert!(before <= sc_timestamp && sc_timestamp <= after);
```

Comment:
This does not add much value to the test. Here we want to test the abort function and see if it can return an error message. So there are no point in adding or mixing with Date.now() here?

Reply:
Well the builtin SC is supposed to test date and it wasn't, Aurélien had some concerns about the return value not being handled properly so I added that

Reply:
I agree the tests should test the built-ins and date is one of them.